--- .Build/vendor/kaystrobach/migrations/Classes/Typo3ConfigurationLoader.php	2025-01-27 17:08:20
+++ .Build/vendor/kaystrobach/migrations/Classes/Typo3ConfigurationLoader-copy.php	2025-01-27 17:09:43
@@ -70,7 +70,7 @@
      */
     private function getPackageMigrationNamespaceAndDirectory(PackageInterface $package): array
     {
-        $autoloadComposerDefinition = $package->getValueFromComposerManifest('autoload');
+        $autoloadComposerDefinition = $package->getValueFromComposerManifest('autoload-dev');

         if (($autoloadComposerDefinition->{'psr-4'} ?? null) instanceof \stdClass) {
             $psr4Namespaces = get_object_vars($autoloadComposerDefinition->{'psr-4'});
